Resources/Prototypes/_Mono/Shipyard/Expedition/tzipora.yml

```
@@ -0,0 +1,41 @@
+- type: vessel
+  id: Tzipora
+  parent: BaseVessel
+  name: MPS Tzipora
+  description: A advanced medical ship, outfitted with cryo, chem, and surgical capabilities.
+  price: 98530
+  category: Large
+  group: Expedition
+  shuttlePath: /Maps/_Mono/Shuttles/Expedition/tzipora.yml
+  guidebookPage: null
+  class:
+  - Expedition
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)